scripts/build-tree-sitter.sh

```diff
@@ -4,98 +4,119 @@ set -euxo pipefail
 
 BASE_PWD="$PWD"
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+LANGUAGE_DATA_DIR="$SCRIPT_DIR/../Sources/tree_sitter_language_resources/LanguageResources"
+
 TMP_BUILD_DIR=$( mktemp -d )
-mkdir $TMP_BUILD_DIR/build
+mkdir -p $TMP_BUILD_DIR/build/{macos,iphoneos,iphonesimulator}
 
-PKG_CONFIG_PATH="$TMP_BUILD_DIR/build/lib/pkgconfig"
-export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+:}${PKG_CONFIG_PATH}:$(pkg-config --variable pc_path pkg-config)
+IPHONEOS_SYSROOT=$(xcrun --sdk iphoneos --show-sdk-path)
+IPHONESIMULATOR_SYSROOT=$(xcrun --sdk iphonesimulator --show-sdk-path)
 
-COMMON_FLAGS="-arch arm64 -arch x86_64 -mmacosx-version-min=10.13"
-LANGUAGE_DATA_DIR="$SCRIPT_DIR/../Sources/tree_sitter_language_resources/LanguageResources"
+MACOS_COMMON_FLAGS="-arch arm64 -arch x86_64 -mmacosx-version-min=10.13"
+IPHONEOS_COMMON_FLAGS="-arch arm64 -miphoneos-version-min=11.0 -fembed-bitcode -isysroot $IPHONEOS_SYSROOT"
```

Comment:
I think the iOS slice suppose to go as low as `armv7s`?

Reply:
I thought so too, but according to some research, it looks like its `arm64`-only as of iOS 11!